src/scenes/Organization/OrganizationForm.js

```diff
@@ -26,21 +27,35 @@ export class OrganizationForm extends Component {
 
   render() {
     return (
-      <View>
-        <FormLabel>Name</FormLabel>
-        <FormInput
-          onChangeText={value => this._onChange('name', value)}
-          ref={ref => (this.nameInput = ref)}
-        />
-        <FormValidationMessage>
-          {this.state.error && <Text>Name cannot be blank</Text>}
-        </FormValidationMessage>
+      <View style={styles.container}>
+        <View style={styles.form}>
+          <FormLabel>Name</FormLabel>
+          <FormInput
+            onChangeText={value => this._onChange('name', value)}
+            ref={ref => (this.nameInput = ref)}
+          />
+          <FormValidationMessage>
+            {this.state.error && <Text>Name cannot be blank</Text>}
```

Comment:
I18n

Reply:
We are not creating currently a international version. Conclusion -> We only support english :)